source/en/components/pageview.md

```diff
@@ -0,0 +1,114 @@
+# PageView component reference
+
+The Page view, Use when you need to switch the entire page.
+
+![pageview-inspector](./pageview/pageview-inspector.png)
+
+Click the 'add component' pageView at the bottom of the **Properties** panel and select 'PageView' from 'add UI component'. You can then add the PageView component to the node.
```

Comment:
Click 'Add Component' button at the bottom .... from 'Add UI Component'. This will add PageView component to the current node.